### ISSUE TITLE
Move project to vala-lang namespace?

### ISSUE THREAD

**Issue by tintou** [freesmartphone/vala-dbus-binding-tool#7]
This project would probably gain to be in the https://github.com/vala-lang namespace? Would you oppose it moving there?

**Comment by mickeyl** [comment]
Not at all! I'm not really working on+with Vala anymore, but I still have a sweet spot for the language that made so many great projects possible. Would love, if it was still useful. Can you create the repository there and I add a move/deprecation notice and archive it here?

**Comment by lw64** [comment]
The only difference to gdbus-codegen I could find is that it also generates asynchronous functions. Is that correct?

**Comment by tintou** [comment]
@lw64 gdbus-codegen does generates C code, here it generates Vala code

**Comment by tintou** [comment]
@mickeyl perfect, we're looking at how we can do it, according to https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#about-repository-transfers we should probably add you as Developer to the vala-lang organization for proper ownership transfer :slightly_smiling_face:

**Comment by lw64** [comment]
> @lw64 gdbus-codegen does generates C code, here it generates Vala code

So it generates a class with a `[DBus]` attribute?

**Comment by mickeyl** [comment]
> @mickeyl perfect, we're looking at how we can do it, according to https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#about-repository-transfers we should probably add you as Developer to the vala-lang organization for proper ownership transfer 🙂

@tintou Sounds like a good idea, yes, please do that.
